packages/grafana-schema/src/scuemata/dashboard/dashboard.cue

```diff
@@ -320,6 +320,7 @@ Family: scuemata.#Family & {
                     if !list.Contains(compose._panelTypes, type) {
                         options: {...}
                         fieldConfig: defaults: custom: {...}
+                        ...
```

Comment:
This makes the base panel structure open for unknown panel types. That means that schema on explicitly stated fields will still be enforced for known and unknown panel types, but unknown types ones can introduce fields wherever they want on the Panel structure.

That will allow legacy angular panels - like heatmap, but also including third party ones, for which we can obviously never write special exceptions - will be able to pass validation.

Could plausibly get rid of the special `#GraphPanel` in the dashboard scuemata, too, though i'll look at that in a follow-up.

Reply:
I like this idea 🚀

Reply:
nice!